lightningnetwork/lnd
# multi: query chan update timestamps

In this PR, we start supporting the timestamps query option in the `query_channel_range` message. If we receive such a message with that bit set then we now populate the timestamps field of the `reply_channel_range` message. 

With this feature we can now ask for the timestamps for the latest channel updates for a channel to be sent along with the SCID. Thus, if we have marked one of the channels in the list as a zombie but we see that the timestamp sent with it make it not a zombie, then we request the channel_update messages for that channel so that we can perhaps awaken the channel. 

### How to test?

See this comment: https://github.com/lightningnetwork/lnd/pull/8030#issuecomment-1808145257

Fixes https://github.com/lightningnetwork/lnd/issues/7989
